crates/ark/src/lsp/completions/sources/composite/call.rs

```diff
@@ -302,8 +302,8 @@ fn completions_from_workspace_arguments(
 #[cfg(test)]
 mod tests {
     use harp::eval::RParseEvalOptions;
-    use tree_sitter::Point;
 
+    use crate::fixtures::point_from_cursor;
```

Comment:
I find tests that use this fixture much easier to read, so I've updated anything I touched to use this.